packages/ruby/examples/metrics-rails/app/controllers/metrics_controller.rb

```diff
@@ -0,0 +1,27 @@
+require 'readme/webhook'
+
+class MetricsController < ApplicationController
+  def index
+    render json: { 'message' => 'hello world' }
+  end
+
+  def post
```

Comment:
what's the point of this route—just to test that things are working?

Reply:
Yeahh, we have a bunch of POST method integration tests e.g. https://github.com/readmeio/metrics-sdks/blob/84776f710019ff814a5c37d1eaa627e6dc510b4a/__tests__/integration-metrics.test.js#L301-L325

Here's the equivalent for express: https://github.com/readmeio/metrics-sdks/blob/84776f710019ff814a5c37d1eaa627e6dc510b4a/packages/node/examples/express/index.js#L31-L33